saleor/webhook/serializers.py

```diff
@@ -16,18 +18,32 @@
 def serialize_checkout_lines(checkout: "Checkout") -> List[dict]:
     data = []
     channel = checkout.channel
+    currency = channel.currency_code
+    included_taxes_in_price = include_taxes_in_prices()
+
     for line_info in fetch_checkout_lines(checkout):
+        line_id = graphene.Node.to_global_id("CheckoutLine", line_info.line.pk)
         variant = line_info.variant
         channel_listing = line_info.channel_listing
         collections = line_info.collections
         product = variant.product
         base_price = variant.get_price(product, collections, channel, channel_listing)
+        unit_price = line_info.line.unit_price
+        price = (
```

Comment:
All prices passed here should be in TaxedMoney. This webhook payload i is generic so we want to have all info explicitly passed in the correct fields.